project_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -38,9 +38,11 @@ def _running_domain(self):
         """Domain to find running timesheet lines."""
         return [
             ("date_time", "!=", False),
-            ("employee_id", "in", self.env.user.employee_ids.ids),
             ("project_id.allow_timesheets", "=", True),
             ("unit_amount", "=", 0),
+            '|',
+            ("employee_id", "in", self.env.user.employee_ids.ids),
+            ("user_id", "=", self.env.user.id),
```

Comment:
Shouldn't searching by `user_id` be enough?
Searching by `employee_id` here seems not necessary to me, since `("employee_id", "in", self.env.user.employee_ids.ids)` is already covered by `("user_id", "=", self.env.user.id)` IMO.

Reply:
Right, changed, thanks